scripts/tests/dns-etc-resolvconf.sh

```diff
@@ -21,9 +21,6 @@ function gateway() {
     docker compose exec -it gateway "$@"
 }
 
-# Wait for client to ping httpbin (CIDR) resource through the gateway
```

Comment:
I may be missing something, but was this test needed or was it just to set a baseline before testing DNS?

Reply:
I saw all the other tests start with that `until ping` pattern so I assumed it's to make sure the tunnel is up, in case Docker's health check is not sufficient

Reply:
There might also be a race condition, even with the ping, where the test script thinks the tunnel is ready but we're actually still prepping DNS or something. That could explain the CI flakiness. I'll check it out.

Reply:
Kk, I'll see if I can get this green.

I don't think we need to up the tunnel before running `curl` -- it would probably be good to test that `curl` works just fine.

We had that `until ping` in there from when we were just using ping as a test itself. We'd give the connection 60 seconds to setup and then fail. So we can probably remove most of those.